<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActorCreationDialog</class>
 <widget class="QDialog" name="ActorCreationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Actor type</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="widget">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Particle effect resource</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="particleEffect"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder>particleEffect</zorder>
      <zorder>label_8</zorder>
      <zorder>pushButton_3</zorder>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Empty</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Skin</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Model</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Geometry</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QComboBox" name="comboBoxSkin"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="geometry">
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Box</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Sphere</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mesh</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditModel"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="boxGeometryGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Box Geometry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="boxHx">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Extents</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QDoubleSpinBox" name="boxHy">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QDoubleSpinBox" name="boxHz">
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Type</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="isStatic">
            <property name="text">
             <string>Static</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="isDynamic">
            <property name="text">
             <string>Dynamic</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QGroupBox" name="sphereGeometryGroup">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="title">
          <string>Sphere Geometry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Radius</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="sphereRadius"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QComboBox" name="actorType">
     <item>
      <property name="text">
       <string>Modelled actor</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Particle effect</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Point light</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ActorCreationDialog</receiver>
   <slot>onModelPick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>ActorCreationDialog</receiver>
   <slot>onSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>geometry</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ActorCreationDialog</receiver>
   <slot>onGeometryChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>ActorCreationDialog</receiver>
   <slot>onParticlePick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>312</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actorType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ActorCreationDialog</receiver>
   <slot>onActorTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>-3</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onModelPick()</slot>
  <slot>onSave()</slot>
  <slot>onGeometryChanged(int)</slot>
  <slot>onParticlePick()</slot>
  <slot>onActorTypeChanged(int)</slot>
 </slots>
</ui>
